app/Http/Controllers/RequestPurchaseOrderController.php

```
@@ -0,0 +1,50 @@
+use App\Http\Requests\UpdateRequestPurchaseOrderRequest;
+use App\Http\Resources\RequestPurchaseOrderDetailedResource;
+use App\Http\Resources\RequestPurchaseOrderListingResource;
+
+class RequestPurchaseOrderController extends Controller
+{
+    /**
+     * Display a listing of the resource.
+     */
+    public function index()
+    {
+        $requestPurchaseOrders = RequestPurchaseOrder::paginate(config('app.pagination.per_page', 15));
```

PHPMD (2.15.0)

Avoid using static access to class '\App\Models\RequestPurchaseOrder' in method 'index'. (Clean Code Rules)

(StaticAccess)